runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java

```diff
@@ -341,18 +342,26 @@ public void makeWorldOverlayHint(Graphics2D graphics, ClueScrollPlugin plugin)
 		}
 	}
 
-//	public static CoordinateClue forItemId(int itemId)
-//	{
-//		for (CoordinateClue clue : CLUES)
-//		{
-//			if (clue.itemId == itemId)
-//			{
-//				return clue;
-//			}
-//		}
-//
-//		return null;
-//	}
+	public static CoordinateClue forItemId(int itemId)
```

Comment:
Works but I hope there is a suggestion on how to do this in a more straightforward way.